src/match/tasks.py

```diff
@@ -32,16 +33,13 @@ def matcher():
             print(f"Match error between {pear}: {error_msg}")
 
 
-schedule, _ = CrontabSchedule.objects.get_or_create(
-    minute="3",
-    hour="18",
-    day_of_week="6",
-    day_of_month="*",
-    month_of_year="*",
-)
+# Commented out until we confirm the algorithm works on an endpoint
+# schedule, _ = CrontabSchedule.objects.get_or_create(
```

Comment:
and let's ideally get an endpoint or two to create/delete these cronjobs at some point so we don't need to change manually : )